tests/jit/jit_models_test.py

```diff
@@ -67,6 +66,15 @@ def small_model_params():
             "n_filters": 32,
             "kernel_size": 32,
         },
+        LSTMTasNet.__name__: {
+            "n_src": 2,
+            "hid_size": 4,
+            "n_layers": 1,
```

Comment:
Better test if there are two layers I guess
```suggestion
            "n_layers": 2,
```